model_compression_toolkit/core/common/quantization/quantization_params_generation/qparams_computation.py

```diff
@@ -46,10 +48,14 @@ def calculate_quantization_params(graph: Graph,
 
     """
 
+    Logger.info(f"Running quantization parameters search. "
+                f"This process might take some time, "
+                f"depending on the model size and the selected quantization methods.")
+
     # Create a list of nodes to compute their thresholds
     nodes_list: List[BaseNode] = nodes if specific_nodes else graph.nodes()
 
-    for n in nodes_list:  # iterate only nodes that we should compute their thresholds
+    for n in tqdm(nodes_list):  # iterate only nodes that we should compute their thresholds
```

Comment:
Add description to tqdm. Something like this: `tqdm(nodes_list, desc="Calculating quantization params")`